db/schema.rb

```diff
@@ -12,527 +12,529 @@
-    t.datetime "created_at",                                       null: false
-    t.datetime "updated_at",                                       null: false
-    t.index ["name"], name: "index_vault_servers_on_name", unique: true, length: { name: 191 }, using: :btree
-  end
-
-  create_table "versions", force: :cascade do |t|
-    t.string   "item_type",                     null: false
-    t.integer  "item_id",                       null: false
-    t.string   "event",                         null: false
-    t.string   "whodunnit"
-    t.text     "object",     limit: 1073741823
+    t.index ["external_id", "deleted_at"], name: "index_users_on_external_id_and_deleted_at"
+  end
+
+  create_table "vault_servers", id: :integer, force: :cascade, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci" do |t|
+    t.string "name", null: false
+    t.string "address", null: false
+    t.string "encrypted_token", null: false
+    t.string "encrypted_token_iv", null: false
+    t.string "encryption_key_sha", null: false
+    t.boolean "tls_verify", default: false, null: false
+    t.text "ca_cert"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["name"], name: "index_vault_servers_on_name", unique: true, length: { name: 191 }
+  end
+
+  create_table "versions", id: :integer, force: :cascade, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci" do |t|
+    t.string "item_type", null: false
+    t.integer "item_id", null: false
+    t.string "event", null: false
+    t.string "whodunnit"
+    t.text "object", limit: 4294967295
     t.datetime "created_at"
-    t.index ["item_type", "item_id"], name: "index_versions_on_item_type_and_item_id", length: { item_type: 191 }, using: :btree
+    t.index ["item_type", "item_id"], name: "index_versions_on_item_type_and_item_id", length: { item_type: 191 }
   end
 
-  create_table "webhooks", force: :cascade do |t|
-    t.integer  "project_id", null: false
-    t.integer  "stage_id",   null: false
-    t.string   "branch",     null: false
+  create_table "webhooks", id: :integer, force: :cascade, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci" do |t|
+    t.integer "project_id", null: false
+    t.integer "stage_id", null: false
+    t.string "branch", null: false
     t.datetime "created_at"
     t.datetime "updated_at"
     t.datetime "deleted_at"
-    t.string   "source",     null: false
-    t.index ["project_id", "branch"], name: "index_webhooks_on_project_id_and_branch", length: { branch: 191 }, using: :btree
-    t.index ["stage_id", "branch"], name: "index_webhooks_on_stage_id_and_branch", length: { branch: 191 }, using: :btree
+    t.string "source", null: false
+    t.index ["project_id", "branch"], name: "index_webhooks_on_project_id_and_branch"
+    t.index ["stage_id", "branch"], name: "index_webhooks_on_stage_id_and_branch"
   end
 
+  add_foreign_key "deploy_groups", "environments"
```

Comment:
idk ... will add a new migration to drop them ... already tried a week ago or so and that did not work out :/